mem/mem/protocol.go

```diff
@@ -549,3 +581,317 @@ func (b GL0InvalidateRspBuilder) Build() *GL0InvalidateRsp {
 	r.RespondTo = b.rspTo
 	return r
 }
+
+func (r *DrainRsp) Meta() *sim.MsgMeta {
+	return &r.MsgMeta
+}
+
+func (b DrainRspBuilder) WithSendTime(
```

Comment:
Please check with @MengyangHe1. I think send time and receive time are no longer required in v4.